app/src/main/res/values/styles.xml

```diff
@@ -96,6 +96,12 @@
     <item name="colorPrimary">@color/colorPrimary</item>
     <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
     <item name="colorAccent">@color/colorPrimary</item>
+    <item name="android:buttonBarNegativeButtonStyle">@style/textAllCaps</item>
+    <item name="android:buttonBarPositiveButtonStyle">@style/textAllCaps</item>
```

Comment:
To get textAllCaps on only the positive and negative buttons for the DialogFragments, I had to use these attributes, but they require API level 21. I don't think there is any other work around. Any suggestions?

Reply:
Is this API 21+ for this particular style item, or for using textAllCaps at all? Ie, do our other textAllApps uses also fail below API 21?

Reply:
API level 21 is for using android:buttonBarNegativeButtonStyle and android:buttonBarPositiveButtonStyle, so it is only for this case.

Reply:
Ack. We should look into an alternative way to solve this (e.g. maybe setting it programmatically).